test/StringProcessing/Parse/regex.swift

```diff
@@ -1,6 +1,9 @@
-// RUN: %target-typecheck-verify-swift -enable-bare-slash-regex -disable-availability-checking
+// Using FileCheck to test the diagnostic columns.
```

Comment:
I believe the verifier supports testing columns now https://github.com/apple/swift/pull/26997

Reply:
Ooooh, I didn't know that. Thank you!